google/cloud/aiplatform/utils.py

```diff
@@ -308,3 +308,37 @@ def __getattr__(self, name: str) -> Any:
             client_info=self._client_info,
         )
         return getattr(temporary_client, name)
+
+
+def sequence_to_list(
+    sequence_of_strings: Sequence[str],
+    delimiter: str = ",",
+    strip_characters: str = " ",
+) -> list:
+    """Given an input sequence, return the sequence as a list.
+
+    Example Usage:
+
+        list = sequence_to_list(sequence_of_strings)
+        # sequence_of_strings = ["example1"]
+        # list = ["example1"]
+
+        # sequence_of_strings = "example2"
+        # list = ["example2"]
+
+        # sequence_of_strings = "example1, example2"
+        # list = ["example1", "example2"]
+
+    Args:
+        input_sequence: Sequence[str]
+            Required. A sequence of strings
+
+    Returns:
+        List[str1, Optional[st2], Optional[str3], ...]
+            A list of string(s) of the input sequence of string(s).
+    """
+
+    if isinstance(sequence_of_strings, list):
+        return sequence_of_strings
+    if isinstance(sequence_of_strings, str):
```

Comment:
This implies that `sequence_of_strings` can be a str, whereas your typehint specifies that it should be a `Sequence[str]`.

See https://docs.python.org/3/glossary.html#term-sequence for more details on what a `Sequence` is.

Although a str is technically a special type of Sequence, I don't think it fits the `Sequence[str]` type hint provided.

It seems like the intention is to have a datatype that can represent both a single str and a sequence of strings, for example for gcs_uris. Short of using an abstract class and two subclasses, I don't see a clean way to do that.

Is it too much to ask users to provide an actual Sequence[str]?

Reply:
This just feels like we're abusing the loose typing of Python. 

I wonder how type-checking systems would respond to this and found this interesting conversation: https://github.com/python/mypy/issues/5090

Reply:
Ooooh, actually `Union[str, Sequence[str]]` be what we want?